net.nokyan.Resources.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id": "net.nokyan.Resources",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-nightly",
```

Comment:
using rust nightly is not recommended and a bad idea at the same time.

Reply:
I'm using rust nightly features such as drain_filter

Reply:
It's using Rust stable now.